webextension/js/popup.js

```diff
@@ -782,6 +786,41 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     });
   },
 
+  showAssignedContainers(assignments) {
+    const assignmentPanel = document.getElementById("edit-sites-assigned");
+    const assignmentKeys = Object.keys(assignments);
+    assignmentPanel.hidden = !(assignmentKeys.length > 0);
+    if (assignments) {
+      const tableElement = assignmentPanel.querySelector("table > tbody");
+      while (tableElement.firstChild) {
+        tableElement.firstChild.remove();
+      }
+      assignmentKeys.forEach((siteKey) => {
+        const site = assignments[siteKey];
+        const trElement = document.createElement("tr");
+        const assumedUrl = `https://${site.hostname}`;
+        trElement.innerHTML = escaped`
+        <td><img class="icon" src="${assumedUrl}/favicon.ico"></td>
+        <td title="${site.hostname}" class="truncate-text">${site.hostname}
+          <img
+            class="pop-button-image delete-assignment"
+            src="/img/container-delete.svg"
+          />
+        </td>`;
+        const deleteButton = trElement.querySelector(".delete-assignment");
+        Logic.addEnterHandler(deleteButton, () => {
+          const identity = Logic.currentIdentity();
+          const userContextId = Logic.userContextId(identity.cookieStoreId);
```

Comment:
We seem to do these 2 lines a lot together? Maybe we need a `Logic.currentUserContextId()` method?